migrations/versions/0134_stats_template_usage.py

```diff
@@ -0,0 +1,65 @@
+"""
+
+Revision ID: 0134_stats_template_usage
+Revises: 0133_set_services_sms_prefix
+Create Date: 2017-11-07 14:35:04.798561
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0134_stats_template_usage'
+down_revision = '0133_set_services_sms_prefix'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('stats_template_usage_by_month',
+    sa.Column('template_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('month', sa.Integer(), nullable=False),
+    sa.Column('year', sa.Integer(), nullable=False),
+    sa.Column('count', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['template_id'], ['templates.id'], ),
+    sa.PrimaryKeyConstraint('template_id', 'month', 'year')
+    )
+    op.create_index(op.f('ix_stats_template_usage_by_month_month'), 'stats_template_usage_by_month', ['month'], unique=False)
+    op.create_index(op.f('ix_stats_template_usage_by_month_template_id'), 'stats_template_usage_by_month', ['template_id'], unique=False)
+    op.create_index(op.f('ix_stats_template_usage_by_month_year'), 'stats_template_usage_by_month', ['year'], unique=False)
+    op.drop_table('notification_statistics')
+    op.alter_column('notification_history', 'international',
```

Comment:
We can't update the model with this as of yet, we need to update the data. There is a server command to do this - but it just ended up on the bottom of the list and hasn't been done as of yet.